controllers/api.js

```diff
@@ -258,8 +264,8 @@ router.post('/payinvoice', async function (req, res) {
         await u.savePaidLndInvoice({
           timestamp: parseInt(+new Date() / 1000),
           type: 'paid_invoice',
-          value: +info.num_satoshis + Math.floor(info.num_satoshis * Paym.fee),
-          fee: Math.floor(info.num_satoshis * Paym.fee),
+          value: +info.num_satoshis + Math.floor(info.num_satoshis * internalFee),
```

Comment:
You could probably rely on the original `Paym.fee` property instead of creating a new global variable, and update what `Paym.fee` returns.

Reply:
Please try and report back ...